packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/add-connector/add-connector.html

```diff
@@ -0,0 +1,91 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_connectors_add_back' | translate }}"
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<!-- error messages -->
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2 ng-bind="$ctrl.availableConnector.name" class="mb-0"></h2>
+        <span class="oui-paragraph">
+            Version: {{$ctrl.availableConnector.version}} | Author:
+            {{$ctrl.availableConnector.author}} |
+            <a
+                ng-href="{{$ctrl.availableConnector.documentationUrl}}"
+                target="_blank"
+                rel="noopener"
+                >docs</a
+            >
+        </span>
+    </div>
+    <oui-button variant="primary" size="l" ng-click="$ctrl.addConnector()">
+        <span data-translate="pci_databases_connectors_add_button"></span>
+    </oui-button>
+</div>
+
+<oui-tabs>
+    <oui-tabs-item
+        ng-repeat="group in $ctrl.availableConnector.configuration.getGroups()"
+        heading="{{group.name}}"
+        subheading="{{group.required ? 'required' : ''}}"
+        variant="promotion"
+    >
+        <div class="row">
+            <div class="col-md-6">
+                <div
+                    ng-if="group.name === 'Transforms'"
+                    class="oui-message oui-message_info"
+                    role="alert"
+                >
+                    <span
+                        class="oui-message__icon oui-icon oui-icon-info-circle"
+                        aria-hidden="true"
+                    ></span>
+                    <p class="oui-message__body">
+                        Transformations enable simple message-at-a-time
+                        modifications. See
+                        <a
+                            href="https://kafka.apache.org/documentation.html#connect_transforms"
+                            target="_blank"
+                            rel="noopener"
+                            >documentation</a
+                        >
+                        for more details.
+                    </p>
+                </div>
+                <div
+                    ng-if="group.name === 'Extra'"
+                    class="oui-message oui-message_info"
```

Comment:
**suggestion**: Prefer using `oui-message` component rather than css class.

Reply:
This has been done on the edit PR which will be merged into this one: https://github.com/ovh/manager/pull/6579/commits/096202bbb90f6a96e19061785069f7c99834ecc4